src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -1139,9 +1161,16 @@ private static byte[] getData(Profile p, int tagSignature) {
      * @throws IllegalArgumentException if a content of the {@code tagData}
      *         array can not be interpreted as valid tag data, corresponding to
      *         the {@code tagSignature}
+     * @throws IllegalArgumentException if the profile being updated has
+     *         {@code isBuiltIn} flag set to true
```

Comment:
isBuiltin is a private variable. You can't reference it in public API javadoc.

What this ought to say is
@throw IllegalArgumentException if this is a profile for one of the built-in pre-defined ColorSpaces, i.e those which can be obtained by calling ICC_Profile.getInstance(int cspace) 

There isn't a general way to get that list - but it is short - only 5 colour spaces, and you could put an @see ColorSpace link

Reply:
Thanks Phil. Updated.